src/config.rs

```diff
@@ -437,10 +459,31 @@ fn get_palette<'a>(
     }
 }
 
+/// Merge two TOML configs by mutating the first argument
+fn merge_toml(base: &mut Value, value: &Value) {
```

Comment:
This could avoid cloning by taking `value` directly by value and using `into_iter()` instead of `iter()`:
```suggestion
fn merge_toml(base: &mut Value, value: Value) {
```